lib/fork.js

```diff
@@ -33,7 +33,7 @@ module.exports = function (args) {
 
 		ps.on('exit', function (code) {
 			if (code > 0 && code !== 143) {
-				reject();
+				reject(new Error(file + ' exited with error code ' + code));
```

Comment:
This was the cause of the unhelpful message reported in #154 (not the cause of that particular failure - just the bad message).